### DISCUSSION TITLE
Duvidas

### DISCUSSION THREAD

**Discussion by Diegolivs** [po-ui/po-angular#1786] [Tire suas dúvidas] [answered]
Bom dia, Tenho umas duvidas sobre o PO-UI mais especificadamente sobre os graficos:
1ª duvida: é possivel deixar o tooltip dos graficos (chart) fixos sem necessidade de ficar passando o mouse por cima?
2ª duvida: nos exemplos das documentações as informações dos eixos que são resultados das gridline não tem numeros quebrados, porem no meu projeto sempre tem numeros quebrados (mesmo todos os valores sendo inteiros) é possivel ajustar para que não haja numeros quebrados?
3ªduvida: semelhantemente a pergunta acima é possivel personalizar essas informações de eixo para que quando forem valores em reais por exemplo utilize formatação dos valores?
4ª duvida: tenho um questionamento anterior ( https://github.com/po-ui/po-angular/discussions/1755 ) sobre como montar o tooltip para ficar personalizado alguem já fez? nas tentativas que fiz geralmente quebra a API ou não monta os graficos.

**Comment by anliben** [comment]
Bom Dia @Diegolivs, tudo bem?

Para simplificarmos a resposta destas questões irei responder que não a todas, atualmente não conseguimos ainda este comportamento utilizando o PO-UI.

Attr.

**Reply by Diegolivs** [reply, marked as answer]
Obrigado. 
Quanto a duvida 2 ajustei manualmente no projeto para que os numeros seja divisiveis sem quebrar o problema vai ser quando o valor maximo passar do estipulado.
